Resources/Prototypes/Tiles/tile_migrations.yml

```diff
@@ -56,3 +56,8 @@
 - type: tileAlias
   id: FloorRGlassAlt
   target: FloorRGlass
+
+# 2024-09-01
+- type: tileAlias
+  id: FloorSnowDug
+  target: FloorSnowGrave
```

Comment:
fix the breakline
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206